docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1497,6 +1479,63 @@ simulants living in military group quarters we will set their employer
 to Military, and ensure that their address and zip code match their
 employer_address and employer_zipcode.
 
+.. _census_prl_perturbation:
+
+2.3.16 Perturbation
+~~~~~~~~~~~~~~~~~~~
+
+When we sample from the ACS PUMS to generate new simulants, we are using the
+empirical joint distribution of all attributes derived from ACS
+variables.
+This allows us to replicate correlations of arbitrary complexity, but also
+causes us to over-fit to sampling variation in the ACS PUMS, which would not
+be present in the full population.
+For example, just because there are no individuals in the ACS PUMS with a
+particular combination of demographic attributes, that does not mean we would
+expect to find 0 such people in the entire US.
+For PRL it is particularly important not to generate simulants that are more similar
+to one another than would be expected in a real population, which
+would make matching unrealistically difficult.
+
+To decrease similarity without assuming total independence between attributes,
+we perturb values at sampling time.
+Specifically, we perturb the age and PUMA columns.
+These are the columns with many possible values, where sampling noise is
+likely to be a significant concern even at the substantial sample size (>15 million)
+of the ACS PUMS.
+
+In different components of the simulation, we sample different entity types from the PUMS:
+households, group quarters people, or non-GQ people (individuals living in households).
+The perturbation process is similar no matter the entity type being sampled.
+
+To perturb PUMA, **50% of the time,** we replace the PUMA with a sample from
+an unconditional PUMA distribution; in other words, we use the PUMA value of
+a second independently sampled entity.
+If we are sampling from a subset of the ACS PUMS, for example recent immigrants
+consistent with a non-reference-person move, **80% of the perturbations**
+(40% of all entities sampled, including those without perturbed PUMA)
+sample a replacement PUMA value from the same subset.
```

Comment:
i think you mentioned these docs aren't yet ready for implementation, but when you end up going over them again for implementation, can you specify which subsets (or what partition?) or ACS PUMS you will want to use?

Reply:
oh wait --- do you mean that:

on sim initialization, 100% of the perturbations sample from the entire state
on immigration, 80% will go to PUMAs that have recent immigrants in the data, and 20% will go to a random PUMA in the state?

Reply:
It's almost that. 80% will go to PUMAs sampled (with weighting) from the PUMA values of recent immigrants **of that move type (e.g. non-reference-person move)**. I will add examples here (and possibly in the initialization and immigration sections above) to make this clearer.